gpustack/k8s/manifests.jinja

```diff
@@ -106,6 +128,8 @@ spec:
               value: {{ config.server_url }}
             - name: GPUSTACK_PORT
               value: '8080'
+            - name: GPUSTACK_GATEWAY_MODE
+              value: 'embedded'
```

Comment:
I have updated the deploy manifests for gpustack-runtime, https://github.com/gpustack/runtime/blob/c637d4138e50c901b761533a49fe63bb834e3979/deploy/manifests/kubernetes.yaml#L98-L129. Please check if it is necessary to update our worker deployment manifest.